prow/cmd/mkpj/BUILD.bazel

```diff
@@ -45,3 +38,8 @@ go_test(
     srcs = ["main_test.go"],
     embed = [":go_default_library"],
 )
+
+go_binary(
+    name = "mkpj",
+    embed = [":go_default_library"],
```

Comment:
I think bazel still want to generate this binary although it shouldn't be used?

Reply:
Yes we definitely want a target to build mkpj, ideally without all the cncf junk that's only relevant to one deployment.